tools/libdeflate/Makefile

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (C) 2022 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libdeflate
+
+PKG_VERSION:=1.12
+PKG_HASH:=ba89fb167a5ab6bbdfa6ee3b1a71636e8140fa8471cce8a311697584948e4d06
+PKG_RELEASE:=1
+
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/ebiggers/libdeflate/archive/refs/tags
```

Comment:
Wonder if it would be better to use git directly and use the release hash
6b5b57116c5b1672a2407aa68f3a49c72f877cb3

Reply:
fixed